src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/ExportClient/OtlpHttpMetricsExportClient.cs

```diff
@@ -33,25 +32,15 @@ namespace OpenTelemetry.Exporter.OpenTelemetryProtocol.Implementation.ExportClie
     internal sealed class OtlpHttpMetricsExportClient : BaseOtlpHttpExportClient<OtlpCollector.ExportMetricsServiceRequest>
     {
         internal const string MediaContentType = "application/x-protobuf";
-        private readonly Uri exportMetricsUri;
 
         public OtlpHttpMetricsExportClient(OtlpExporterOptions options, HttpClient httpClient)
-            : base(options, httpClient)
+            : base(options, httpClient, "v1/metrics")
```

Comment:
Not sure of what is the preference for literals used in a single place, happy to add the corresponding constant if that is the preference.

Reply:
+1 to making it a const.